utils/docs/man/man1/atomic-openshift-installer.1.asciidoc.in

```diff
@@ -0,0 +1,167 @@
+atomic-openshift-installer(1)
+=============================
+:man source:   atomic-openshift-utils
+:man version:  %VERSION%
+:man manual:   atomic-openshift-installer
+
+
+NAME
+----
+atomic-openshift-installer - Interactive OpenShift Container Platform installer
```

Comment:
Maybe set the initialism here to be used in the rest of the page:

"Interactive OpenShift Container Platform (OCP) installer"